packages/db/src/artifacts/test/index.ts

```diff
@@ -0,0 +1,245 @@
+import fs from "fs";
+import path from "path";
+import { TruffleDB } from "@truffle/db";
+import { shimBytecode } from "@truffle/workflow-compile/shims";
```

Comment:
This is going to conflict with the PR just merged changing the compilation interface (psst, [this one](https://github.com/trufflesuite/truffle/pull/3252)).

The shims got moved to `compile-common` and have a different api now. [Here, check it out!](https://github.com/trufflesuite/truffle/tree/develop/packages/compile-common/src/shims)

Reply:
@eggplantzzz so now I would import `forBytecode` from `compile-common` instead? Is there more that has changed that I should keep an eye out for? I may fix this as a quick follow-on after this branch gets merged but was wondering if there is more to it.

Reply:
Merged changes to account for new `compile-common` stuff!